src/Credentials.js

```diff
@@ -22,6 +22,9 @@ export default class Credentials {
     if (params.requested) {
       payload.requested = params.requested
     }
+    if (params.verified) {
```

Comment:
What is the reasoning for params.verified? and how does the response from mobile differ in these cases?